cli/test/README.md

```diff
@@ -86,34 +86,50 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec[]`|`ERROR: Participant builtin discovery config <duration>: 'lease' duration type <nanosec> attribute must not be keyed []`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec.other`|`ERROR: Participant builtin discovery config <duration>: 'lease' duration type <nanosec> attribute must be FINAL element`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec arg`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec 100`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec infinite`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec other`|`ERROR: value 'other' does not match any member types of the union`|N/A|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.nanosec -h`|N/A|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements`|`ERROR: set command for Participant builtin discovery config <duration>: 'announcements' duration type expects at least 1 arguments and received 0`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements[]`|`ERROR: Participant builtin discovery config <duration>: 'announcements' duration type must not be keyed []`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements arg1 arg2 arg3`|`ERROR: set command for Participant builtin discovery config <duration>: 'announcements' duration type expects at most 2 arguments and received 3`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements arg1`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements arg1 arg2`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.announcements infinite`|N/A|N/A|
```

Comment:
Please, apply to the lease `announcements` the same suggestions as in the `lease duration`.